tfhe/benches/integer/bench.rs

```diff
@@ -1983,32 +1983,32 @@ mod cuda {
 
     criterion_group!(
         default_cuda_ops,
-        cuda_neg,
-        cuda_sub,
-        cuda_unsigned_overflowing_sub,
-        cuda_unsigned_overflowing_add,
+        //cuda_neg,
```

Comment:
What's the motivation here? Don't we care about these benchmarks?

Reply:
Oh wow good catch I forgot to remove this commit in the rebase. Will revert this change.